### PR TITLE
[feature] New tag selector component

### PR DESCRIPTION
**Pull request by alex-deaconu**
Component used to select tags
Added unit and e2e test coverage

@olegrise please review. Thanks!

### CODE REVIEW

File: test/unit/tag-selector/ctr-tag-lookup.tests.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+      $scope.selectTag(tagsList[1]);
+      
+      $scope.apply();
+    });
+  });
+  
+  describe('cancel: ',function(){
+    it('should cancel',function(){
+      $scope.cancel();
+    });
```
Comment (olegrise) [inline test/unit/tag-selector/ctr-tag-lookup.tests.js]: @alex-deaconu does this test ever fail? How do you know if `$scope.cancel();` function did what it suppose to do?

Comment (alex-deaconu) [inline test/unit/tag-selector/ctr-tag-lookup.tests.js]: @olegrise no. Good point, I will add a status String to check. Will let you know once that's ready. Thanks.

Comment (alex-deaconu) [inline test/unit/tag-selector/ctr-tag-lookup.tests.js]: @olegrise fixed by adding a `modalStatus` String to verify which action was called. Thanks.

**Comment by alex-deaconu** [comment]
Also fixed issue with `scope` attribute in directive.

**Comment by olegrise** [comment]
:+1:

**Pull request merged**